### DISCUSSION TITLE
Installed the internet repo instead of air gap in a distributed environment... should we reinstall?

### DISCUSSION THREAD

**Discussion by richardgiannetti8312** [Security-Onion-Solutions/securityonion#6296] [Unsupported Versions] [answered]
Hi,

We recently installed SO with a distributed deployment and selected internet instead of air gap.  Should we do reinstallation or can we make changes to configuration to set everything to air gap?  Will changes need to be made on all nodes or just the master?  Will there be any issues if we just leave it as internet even though we don't have internet connectivity?

Thanks,
Rich

**Comment by dougburks** [comment, marked as answer]
I'm not sure that we've tested converting an Internet connected deployment to airgap. It's probably best just to do a reinstallation in airgap mode using our upcoming 2.3.90 release:
https://github.com/Security-Onion-Solutions/securityonion/projects/26
